tools/makerootfs.sh

```diff
@@ -26,7 +26,7 @@ do_image() {
   : > "$IMAGE"
   # sbom disabled for now; will be re-enabled later
```

Comment:
This (and the other) comments like this can be removed now....

Reply:
perhaps you can revert commit https://github.com/lf-edge/eve/commit/03357a6e3e178b292ce50cbad50ebcfed12c97df and add the other changes on top of it....

Reply:
:point_up: @deitch , not sure if you saw these....

Reply:
Oh, I missed that note. Thank you.